fastlane/Fastfile

```diff
@@ -580,23 +580,27 @@ platform :ios do
       notify_testers: false
     )
 
-    # Install SentryCLI prior to trying to upload dSYMs
-    sh('curl -sL https://sentry.io/get-cli/ | bash')
-
     sentry_upload_dsym(
       auth_token: get_required_env('SENTRY_AUTH_TOKEN'),
       org_slug: 'a8c',
       project_slug: 'wordpress-ios',
       dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
     )
 
+    return if ENV['BUILDKITE_PULL_REQUEST'].nil?
```

Comment:
Have you considered moving this check to the start of the lane? Is there a use case in which we would run all the above not on a pull request?

Reply:
> Is there a use case in which we would run all the above not on a pull request
I don't know of one, but I also can't entirely rule it out (for instance, @startuptester had requested the ability to create an installable build of `develop` on-demand from a webhook, which I _think_ could call this action and "just work")

Reply:
👍